database/db.go

```diff
@@ -0,0 +1,139 @@
+package database
+
+import (
+	"database/sql"
+
+	_ "github.com/mattn/go-sqlite3" // Import the SQLite driver
```

Comment:
```suggestion
	_ "github.com/mattn/go-sqlite3"
```

Reply:
avoid "comment-code redundancy"

Reply:
I removed the other unnecessary comments, however this is required by the linter (blank import)